### ISSUE TITLE
error updating to elison Unhandled exception: [Enter a title]

### ISSUE THREAD

**Issue by potzik** [remyroy/CDDA-Game-Launcher#427]
* Description: [Enter what you did and what happened]
* Version: 1.4.5
* OS: Windows-10-10.0.18362-SP0 (64-bit)
* Type: `<class 'FileNotFoundError'>`
* Value: [WinError 3] The system cannot find the path specified: 'C:\\Users\\Rustam\\AppData\\Local\\Programs\\CDDA Game Launcher\\cdda\\previous_version\\font'
* Traceback:
```
  File "cddagl\ui\views\main.py", line 2339, in copy_next_soundpack
  File "cddagl\ui\views\main.py", line 2511, in post_extraction_step3
  File "cddagl\ui\views\main.py", line 2392, in preserve_custom_fonts

```

**Comment by remyroy** [comment]
Thanks for the quick report. Seems like some bug is still present in there. Let me try to fix that quickly.

**Comment by remyroy** [comment]
Fixed with 7f754770503297308aa5832e0c52972775fed936 . It will be part of the new release in a few minutes.
